api/app/schemas/morecast_v2.py

```diff
@@ -0,0 +1,53 @@
+""" This module contains pydantic models for Morecast v2"""
+
+from enum import Enum
+from pydantic import BaseModel
+
+
+class ModelChoice(str, Enum):
+    """ Enumerator for all valid forecasted value types """
+    GDPS = 'GDPS'
+    GFS = 'GFS'
+    HRDPS = 'HRDPS'
+    RDPS = 'RDPS'
+    MANUAL = 'MANUAL'
+
+
+class ForecastedTemperature(BaseModel):
+    """ Forecaster chosen temperature """
+    temp: float
+    choice: ModelChoice
+
+
+class ForecastedRH(BaseModel):
+    """ Forecaster chosen rh """
+    rh: float
+    choice: ModelChoice
+
+
+class ForecastedPrecip(BaseModel):
+    """ Forecaster chosen 24-hour precipitation mm """
+    precip: float
+    choice: ModelChoice
+
+
+class ForecastedWindSpeed(BaseModel):
+    """ Forecaster chosen wind speed """
+    wind_speed: float
+    choice: ModelChoice
+
+
+class ForecastedWindDirection(BaseModel):
+    """ Forecaster chosen wind direction """
+    wind_direction: float
+    choice: ModelChoice
+
+
+class MorecastForecast(BaseModel):
+    """ Forecasted daily """
+    station_code: int
```

Comment:
Date, date offset? We need this to identify what date this forecast is for.


Figure out how WF1 
- timestamp, in millis?
- station code or station id/uuid?

Reply:
Added timestamp field in: https://github.com/bcgov/wps/pull/2643/commits/8c47243c3cbeec6a786b69548b89f29272d25add

Will assume for now that API accepts station code instead of id/uuid